src/creature.cpp

```diff
@@ -102,6 +102,12 @@ void Creature::reset()
     reset_bonuses();
     reset_stats();
 }
+
+void Creature::bleed() const
```

Comment:
This looks really unnecessary. It doesn't really fix any code duplication since it would be used in very few places and it already managed to gain a potential bug: `bloodType` can return `fd_null`, which `add_splatter` will then use to create a field of nothing.